resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -123,11 +115,9 @@
 			"r_c seems to be oblivious to m_c glaring at {PRONOUN/r_c/object} from across camp.",
 			"m_c is sharpening {PRONOUN/m_c/poss} claws near r_c.",
 			"r_c notices that m_c knows a lot about the Dark Forest.",
-			"m_c says a fun fact about the Dark Forest to r_c.",

```

Comment:
Removed

Reply:
Thought this one was too similar to the one right above it!